File: problems/58/solution.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        if not s:
            return 0
        
        l = 0
        s = s.strip()
        i = len(s)
        
        while i > 0:
            i -= 1
            
            if s[i] != ' ':
                l += 1
            else:
                break
        
        return l

File: problems/58/solution.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int pos = s.size() - 1, len = 0;
        
        while (pos >= 0 && s[pos] == ' ') {
            pos--;
        }
        
        while (pos >= 0 && s[pos] != ' ') {
            pos--;
            len++;
        }
        
        return len;
    }
};